BUG-BOUNTY.md

```diff
@@ -0,0 +1,119 @@
+# ERC-721 Token Reference Implementation Bug Bounty
+
+*This documents 0xcert's bug bounty process and how you get paid for finding issues with the ERC-721 Token Reference Implementation.*
+
+## Leaderboard
+
+First, let's give credit to the security researchers that reported bugs!
+
+| Bug report                                                   | Severity | Researcher    |
+| ------------------------------------------------------------ | -------- | ------------- |
+| #51                                                          | Low      | @dafky2000    |
+| [`a03db28`#commitcomment-29251894](https://github.com/0xcert/ethereum-erc721/commit/a03db28eb22ad5a3fc17c707152db46166068a1d#commitcomment-29251894) | Low      | @matbest      |
+| #87                                                          | Low      | @mg6maciej    |
+| #91                                                          | Low      | @mudgen       |
+| #97                                                          | Low      | @mg6maciej    |
+| #100                                                         | Medium   | @mg6maciej    |
+| #102                                                         | Low      | @coburncoburn |
+| #106                                                         | High     | @mg6maciej    |
+
+## Sponsors
+
+**Sponsor this bug bounty if you support ERC-721**. This means you will commit to pay researchers that demonstrate a problem. Contact us at [bounty@0xcert.org](mailto:bounty@0xcert.org) if interested. Thank you.
+
+Sponsors: ![0xcert sponsor](https://img.shields.io/badge/0xcert-50,000%20USD-red.svg)![William Entriken sponsor](https://img.shields.io/badge/William%20Entriken-+1,000%20 USD-blue.svg)
+
+Total bounty: ![ERC-721 Bug Bounty](https://img.shields.io/badge/ERC--721%20Bug%20Bounty-51,000%20USD-purple.svg)
+
+---
+
+## Scope of this bounty program
+
+This bounty is open for an unlimited time. A previous limited-time bounty program was:
+
+* [Round 1](https://github.com/0xcert/ethereum-erc721/issues/46) — **2018-05-16 at 00:01 CET** to **2018-07-16 at 23:59 CET**
+
+Help us find any problems with this contract and with ERC-721 in general. This bounty program's function scope includes:
+
+- Overflow or break parts of the implementation
+- Steal ownership of a token
+- Give a token to somebody else and double spend it or revert it back to your control
+- Any undocumented and unintuitive behavior
+- Typos
+- Style guide violations
```

Comment:
This is not consistent with the risk table and rewards since this violations fall under notable. Should we really count typos and whitespace errors toward reward payouts?

Reply:
Notable is "no cash payout" but read further, they still get a tee shirt and name listed on the leaderboard.

Reply:
If January comes and we gave out 20 tee shirts because typos were fixed I will be happy, and embarrassed. But mostly happy.